integration/internal/container/ns.go

```diff
@@ -0,0 +1,20 @@
+package container
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/client"
+	"gotest.tools/v3/assert"
+	is "gotest.tools/v3/assert/cmp"
+)
+
+// Gets the value of the specified namespace of a container
+func GetContainerNS(ctx context.Context, t *testing.T, client client.APIClient, cID, nsName string) string {
+	res, err := Exec(ctx, client, cID, []string{"readlink", "/proc/self/ns/" + nsName})
```

Comment:
While at it, perhaps make this helper?

```suggestion
	t.Helper()
	res, err := Exec(ctx, client, cID, []string{"readlink", "/proc/self/ns/" + nsName})
```

Reply:
fixed